tests/test_main.py

```diff
@@ -185,7 +185,7 @@ def test_humanitarian_needs(self, configuration, folder, pipelines):
         count = session.scalar(
             select(func.count(DBHumanitarianNeeds.resource_hdx_id))
         )
-        check.equal(count, 139085)
+        check.equal(count, 44938)
```

Comment:
Maybe add a comment that the test file only has 3 countries

Reply:
Good idea!